core/hydromet_stratified.py

```diff
@@ -383,6 +381,169 @@ def Scenarios_low_and_high_S(df_runoff_SR1: pd.DataFrame, mu: float, GEV_paramet
                                    'Avg. S (Upper 50%)', 'Rainfall (Upper 50%)'])
     return df_SR2
 
+def precip_hyetograph_nrcs(df : pd.DataFrame) -> pd.DataFrame:
+    '''This function takes the dataframe precip table extracted from NOAA Atlas 14 and calculates
+    the nested hyetograph for storm events classified by recurrence intervals. The function first 
+    retreives the ratio of rainfall, and incremental intensity and then proceeds to get ratio, slope, and slope difference\
+    and finally fits parabolic curvefrom 0 to 9 hours that passes through the ratios at 0
+    , 6, and 9 hours. The function then fits curves for the remaining data until 12 hours.
+    '''
+    time_range = {'time':np.arange(start =0, stop = 241,step = 1)}
```

Comment:
This is a lot of hard coding. It looks like this method can only be used for 24 hour durations. If that is the goal then I understand, otherwise this may want to be updated so that the variables can be passed from the notebook or read in from a json file.

Reply:
Comment that function/code is limited to 24 hours and needs to be updated in the future to be flexible for different storm durations.

Reply:
Done, Thanks!